apps/modernization-ui/src/apps/page-builder/page/management/edit/subsection/Subsection.tsx

```diff
@@ -45,7 +48,14 @@ export const Subsection = ({ subsection, onAddQuestion }: Props) => {
                 fetch(page.id);
             });
         } else {
-            console.log('delete question NYI', id);
+            PageQuestionControllerService.deleteQuestionUsingDelete({
+                authorization: authorization(),
+                page: page.id,
+                questionId: Number(id)
+            }).then(({ message }) => {
```

Comment:
Issue(blocking): The `message` seems to be throwing an undefined error, might be better to remove it all together.